linkerd/admin/src/main/scala/io/buoyant/linkerd/admin/LinkerdAdmin.scala

```diff
@@ -36,6 +36,7 @@ class LinkerdAdmin(app: App, linker: Linker, config: LinkerConfig) extends Admin
     "/delegator" -> new DelegateHandler(AdminHandler, dtabs, interpreterForRouter),
     "/delegator.json" -> new DelegateApiHandler(interpreterForRouter),
     "/metrics" -> MetricsHandler,
+    "/help" -> new HelpPageHandler,
```

Comment:
hmmmm that's a good idea, though I wanted to use the AdminHandler to avoid copying the navbar html, as well as the stylesheets to also be inserted into the html for this page...